stories/components/MdInput.stories.tsx

```diff
@@ -7,18 +8,46 @@ export default {
   title: 'Components/MdInput'
 }
 
-export const Basic: React.FC = () => <MdInput />
+const Template: Story = (args: { presetValue?: string }) => {
```

Comment:
This should become the new way we use to declare stories. It's much better than our current way of re-declaring the component for each story.